cfme/automate/buttons.py

```diff
@@ -183,6 +185,7 @@ class ButtonDetailView(AutomateCustomizationView):
     type = SummaryFormItem("Object Attribute", "Type")
 
     show = SummaryFormItem("Visibility", "Show")
+    user_roles = SummaryFormItem("Visibility", "User Roles")
```

Comment:
TODO:

This view should be using `SummaryForm`, which is parametrized and will really reduce the number of individual widgets we have to define here.

Reply:
we will...